features/facebookImport/test/ministory/report-metadata.test.js

```diff
@@ -97,22 +93,6 @@ describe("Report metadata analysis", () => {
             preferedLanguage
         );
     });
-
-    it("has correct polyPod version in analysis", async () => {
-        expect(reportData.polyPodVersion).toBe(MOCKED_POD_RUNTIME_VERSION);
-    });
```

Comment:
I thought these tests were actually kinda useful - wouldn't it be rather easy to bring `MOCKED_POD_RUNTIME` and `MOCKED_POD_RUNTIME_VERSION` back? If it was a true mock, it'd even allow us to _set_ the desired values here, but I think that's going too far maybe.

Reply:
Well, they're not testing any business logic, so they're not terribly useful. But I guess they're good to freeze the interface, as I always say. I'll bring them back.

Reply:
The problem is that I don't find the source of truth for these values. And this is only importing the constants, not the `MockPod`, so it's not really possible to set these values in the pod. I can create a mockpod with mock values and see if they're equal but, again, don't see any value in that.
What I'm going to do is to test if values in `reportData` and `analysisData` are the same. Kinda halfway there, I know, but it avoids adding complexity by adding dependencies and keeps the same spirit.